data/scripts/fetch_wikipedia.py

```
@@ -0,0 +1,103 @@
+def fetch(config: str, split: str, out_dir: Path, limit: int | None, records_per_file: int) -> list[str]:
+    out_dir.mkdir(parents=True, exist_ok=True)
+    files = _parquet_urls(config, split, limit)
+    if not files:
+        raise ValueError(f"no parquet files found for {config}/{split}")
+    print(f"Found {len(files)} parquet shard(s); extracting text column ...")
+    written: list[str] = []
+    for i, entry in enumerate(files, 1):
+        url = entry["url"]
+        shard = entry["url"].rsplit("/", 1)[-1].rsplit(".", 1)[0]
+        print(f"[{i}/{len(files)}] {shard} ({entry.get('size', 0) // 1048576} MB) ...", flush=True)
+        tmp = Path("/tmp") / f"wiki-{shard}.parquet"
```

Ruff (0.16.1)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)